test/test-metrics.js

```diff
@@ -0,0 +1,143 @@
+const { start } = require('..');
+const test = require('tape');
+const request = require('supertest');
+
+const errHandler = t => err => {
+    t.error(err);
+    t.end();
+};
+
```

Comment:
It might be good to add at least one test asserting right value was return, e.g. try invoke a function `/` twice and assert that invocation count in metrics is two.